### PR TITLE
vote: CabinfeverB as pd reviewer

### PR DESCRIPTION
**Pull request by rleungx**

### CODE REVIEW

File: votes/0165-CabinfeverB-as-pd-reviewer [hunk 1]
```diff
@@ -0,0 +1,18 @@
+# A Vote for CabinfeverB as PD Reviewer
```
Comment (skyzh) [inline votes/0165-CabinfeverB-as-pd-reviewer:1]: missing `.md` for this file name?

Comment (rleungx) [reply votes/0165-CabinfeverB-as-pd-reviewer]: LOL, you caught me.

**Review by AndreMouche — APPROVED** [review]
LGTM

**Review by JmPotato — APPROVED** [review]
Congratulations! 🎉

**Review by nolouch — APPROVED** [review]

**Pull request merged**